src/main/java/com/infoshareacademy/service/GameService.java

```diff
@@ -0,0 +1,97 @@
+package com.infoshareacademy.service;
+
+import com.infoshareacademy.model.Game;
+import com.infoshareacademy.model.Questions;
+import com.infoshareacademy.model.User;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+public class GameService {
+    private static Game game = new Game();
+
+    public static void start(List<Questions> questions, User user) throws InterruptedException {
+        game.setUser(user);
+        game.setQuestions(questions);
+
+        System.out.println("START NEW GAME: ");
+        displayQuestions(questions);
+        finalResult(questions);
+    }
+
+    private static void displayQuestions(List<Questions> questionsList) throws InterruptedException {
+        int questionNumber = 0;
+        game.setAllUserAnswers(new boolean[questionsList.size()][4]);
+        for (Questions quest : questionsList) {
+            questionNumber++;
+            // display the questions
+            System.out.println();
+            System.out.print(questionNumber + ". Pytanie: ");
+            System.out.println(quest.getQuestionText());
```

Comment:
Można to rozpisać w bardziej elegancki sposób. Definiowanie zmiennej questionNumber i inkrementowanie jej za pomocą ++ też jest zbędne. np przez użycie fori